doc/samples/SqlDataReader_DataDiscoveryAndClassification.cs

```diff
@@ -0,0 +1,151 @@
+﻿// <Snippet1>
+using System;
+using Microsoft.Data.SqlClient;
+using Microsoft.Data.SqlClient.DataClassification;
+
+class Program
+{
+    // Name of the temporary table created for this sample program.
+    static string tableName = "SQLCLIENT_DATA_DISCOVERY_CLASSIFICATION";
+
+    public static void Main()
+    {
+        // To avoid storing the connection string in your code, you can retrieve it from a configuration file. 
+        string connectionString = "Data Source=localhost; Integrated Security=true; Initial Catalog=AdventureWorks;";
+
+        // Open a connection to the AdventureWorks database.
+        using SqlConnection connection = new SqlConnection(connectionString);
+        connection.Open();
+
+        using SqlCommand command = connection.CreateCommand();
+        try
+        {
+            // Check if the target SQL Server supports Data Discovery and Classification.
+            if (DataClassificationSupported(command))
+            {
+                // Create the temporary table and retrieve its Data Discovery and Classification information.
+                CreateTable(command);
+                RunTests(command);
+            }
+        }
+        finally
+        {
+            // Drop the temporary table.
+            DropTable(command);
+        }
+    }
+
+    /// <summary>
+    /// Verifies if SQL Data Discovery and Classification feature is available on the target server.
+    /// </summary>
+    /// <param name="command">The SqlCommand to work with.</param>
+    /// <returns>True if the target SQL Server supports the feature and false otherwise.</returns>
+    public static bool DataClassificationSupported(SqlCommand command)
```

Comment:
You can pass SqlConnection into the methods and use a new instance of SqlCommand for each method.